gui/src/2D/controls/structuredTextBlock.ts

```diff
@@ -0,0 +1,779 @@
+import { Observable } from "babylonjs/Misc/observable";
+import { Measure } from "../measure";
+import { ValueAndUnit } from "../valueAndUnit";
+import { Control } from "./control";
+import { TextWrapping } from "./textBlock";
+import { RegisterClass } from "babylonjs/Misc/typeStore";
+import { Nullable } from "babylonjs/types";
+import { serialize } from 'babylonjs/Misc/decorators';
+import { ICanvasRenderingContext , ICanvasGradient } from 'babylonjs/Engines/ICanvas';
+import { Engine } from 'babylonjs/Engines/engine';
+
+export interface StructuredTextPart {
```

Comment:
Cab you extract these interfaces into separate files?

Reply:
Ok, where should I put it? `gui/src/2D/controls` or `gui/src/2D`?

Reply:
Also it seems that you are prefixing interfaces with a `I`, so should I rename them to `IStructuredText` and `IStructuredTextPart` and name the file `iStructuredText.ts`?

Reply:
`I` prefix & putting them directly in the `gui/src/2D/controls` folder should be ok :)

Reply:
that was what I tried to say :)